diff --git a/pkgs/by-name/ar/arcan/package.nix b/pkgs/by-name/ar/arcan/package.nix
index ce4fc7256b77..49f0abf9f752 100644
--- a/pkgs/by-name/ar/arcan/package.nix
+++ b/pkgs/by-name/ar/arcan/package.nix
@@ -57,12 +57,12 @@ let
   allSources = {
     letoram-arcan = {
       pname = "arcan";
-      version = "0.6.2.1-unstable-2023-11-18";
+      version = "0.7-unstable";
       src = fetchFromGitHub {
         owner = "letoram";
         repo = "arcan";
-        rev = "0950ee236f96a555729498d0fdf91c16901037f5";
-        hash = "sha256-TxadRlidy4KRaQ4HunPO6ISJqm6JwnMRM8y6dX6vqJ4=";
+        rev = "cb72ac6bba5304749b688af94b265b23cc54a216";
+        hash = "sha256-YHycWq1+M9BT+5797I3uCbELl2POndiXoqKZroB4Bvs=";
       };
     };
     letoram-openal-src = fetchFromGitHub {
